horizon/pdp.py

```diff
@@ -99,14 +99,14 @@ def __init__(self):
             raise SystemExit(GUNICORN_EXIT_APP)
 
         if not remote_config:
-            logger.warning(
-                "Could not fetch config from cloud control plane, reverting to local config!"
-            )
-        else:
-            logger.info("Applying config overrides from cloud control plane...")
-            apply_config(remote_config.opal_common or {}, opal_common_config)
-            apply_config(remote_config.opal_client or {}, opal_client_config)
-            apply_config(remote_config.pdp or {}, sidecar_config)
+            logger.critical("No cloud configuration found. Exiting.")
+            raise SystemExit(GUNICORN_EXIT_APP)
```

Comment:
I know this code was here before, but doesn't get_remote_config return a remote config ? it's return value is Optional ?

Reply:
It returns None if fetching fails due to any issue other than invalid api key (in that case it would raise).
Previously it didn't make much sense to treat fetching failure as an "expected" flow, but I guess with the "offline mode" it makes more sense.